openpathsampling/features/numpydocparse.py

```diff
@@ -0,0 +1,145 @@
+class NumpyDocParser(object):
```

Comment:
Actually, that was the point. Sphinx will open the package and read the `__doc__` property and put that into the documentation. This was copied from my build html page
```
Snapshot([engine, configuration, momentum, ...])	The standard MDSnapshot supporting coordinate, velocities and box_vectors
```

Since I was on the engine module I have not updated the docs yet and there is something wrong with autosummary. But I think that sphinx can really use the generated docstring. 

If that is true then for sphinx there is no distinction between generated and hard-written docs as long as the docs are generated on import.